khal/settings/utils.py

```diff
@@ -49,6 +50,13 @@ def is_timezone(tzstring):
         raise VdtValueError("Unknown timezone {}".format(tzstring))
 
 
+def is_timedelta(string):
```

Comment:
Why is this called `is_timedelta` if it doesn't return a boolean? Something like `parse_timedelta` would be more appropriate.